auth0/node-samlp
# feat: add support for per-participant bindings during SLO

By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Add support for participant-specific bindings for SLO. The library will use the correct binding if it is specified in the SessionParticipant structure.

### Testing

1. Set up SAML SLO with multiple participants.
2. Add a `binding` field in the participant structure with mixed values.
3. Trigger SAML SLO and observe that each participant gets called using its specific binding.

See the added tests for details.


### Checklist
